.gitignore

```diff
@@ -8,3 +8,5 @@ TODO
 .idea/
 elasticsearch.iml
 !/vars/RedHat.yml
+.DS_store
```

Comment:
I would recommend sticking this in your local [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore). Much easier than having to add it to all projects.